### PR TITLE
Enable ReadTheDocs

### PR DESCRIPTION
**Pull request by cjgratacos**
Task: #120 

* [x] Ready for review
* [x] Ready for merge

#### Adds integration with ReadTheDocs.

The given DNS by ReadTheDocs are:

- [contentacms.readthedocs.io](contentacms.readthedocs.io)
- [contentacms.rtfd.io](contentacms.rtfd.io)
---
#### Note:
- ReadTheDocs is configured to have branch `8.x-1.x` to be the default latest, thus making the DNS given on the top not to work correctly until the PR gets merged. To see the documentation generated by this branch, go [here](http://contentacms.readthedocs.io/en/feature-120-readthedocs/). 
- There is still work to be done. Adding more documentation and fixing the one we currently have.

### CODE REVIEW

File: docs/license.md [hunk 1]
```diff
@@ -0,0 +1,636 @@
+# GNU GENERAL PUBLIC LICENSE
+Version 3, 29 June 2007
```
Comment (e0ipso) [inline docs/license.md]: We cannot have a license file in here. When we move the distro to Drupal.org the packager will add a license file. That license will conflict with this one then.

File: docs/development.md [hunk 2]
```diff
@@ -7,9 +7,9 @@
 - [npm](https://nodejs.org/en/download/)
 
 ```bash
-git clone git@github.com:drupal-http-apis/contenta_jsonapi.git
-cd contenta_jsonapi
-sh scripts/development/build-contenta_jsonapi.sh
+$ git clone git@github.com:drupal-http-apis/contenta_jsonapi.git
```
Comment (e0ipso) [inline docs/development.md]: Will those `$ ` impact the ability to copy and paste?

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Site Information
+site_name: ContentaCMS
+repo_url: https://github.com/contentacms/contenta_jsonapi/
+repo_name: contentacms/contenta_jsonapi
+site_description: ContentaCMS Documentation
```
Comment (e0ipso) [inline mkdocs.yml]: `Contenta CMS Documentation`

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# ContentaCMS
```
Comment (e0ipso) [inline docs/index.md]: `# Contenta CMS`

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+
+## Introduction
+
+## More Comming Soon!
```
Comment (e0ipso) [inline docs/index.md]: What do you think of

```
# Contenta CMS

Contenta is an API-First Drupal distribution. It's all about the content. Learn more at http://contentacms.org.

## [](#goals)Goals
One of the conclusions of DrupalCon Baltimore was that Drupal 8 has **impressive tools to build a decoupled application**. However, assembling all the tools together and setting them up correctly can be a little bit daunting. On top of that, once you have everything set up you need to figure out the ins and outs of your front-end project.

With this in mind, Contenta CMS is born to _make your content happy_. Contenta CMS is the response of **the community** to build an API-First Drupal distribution. The goals of Contenta are simple: provide a standard platform that is API ready along with demo content and example front-end applications. In summary, Contenta intends to ease the pain of using, or simply trying, decoupled Drupal.

## [](#demos)Demos
An integral part of Contenta is to provide an out of the box content model and demo content ready for you to start working with it. The demo chosen by the community is a [recipe magazine](https://www.drupal.org/node/2818741).

With this in mind, every demo consumer will implement this recipe magazine site with the goal in mind to compare the different implementations.

### [](#example-front-ends)Example Consumers

Choosing your own front-end technology is one of common reasons to choose a decoupled approach. In a decoupled architecture you can also have multiple front-ends at the same time. Contenta gives you examples on how to build a consumer application in different technologies.

We will implement the [same wireframes](https://www.drupal.org/node/2818741#comment-12122841) as the Out of the Box Initiative, so all the demo applications can be compared. Even though all applications will have the same structure, each one will choose their look and feel. Angular may use [material design](https://material.io/guidelines/material-design/introduction.html), elm may use [bootstrap](http://getbootstrap.com/), etc.

| Front-end | Status       | Example | Code   |
|:----------|:-------------|:--------|:-------|
| Angular   | Running demo | [Website](https://contenta-angular.firebaseapp.com/) | [Repo](https://github.com/contentacms/contenta_angular) |
| Elm       | Running demo | [Website](https://contenta-elm.firebaseapp.com/) | [Repo](https://github.com/contentacms/contenta_jsonapi__elm) |
| Ember     | Need help    | Not yet | [Repo](https://github.com/contentacms/contenta_ember) |
| Ionic     | Running demo | Not yet | [Repo](https://github.com/contentacms/contenta_ionic) |
| React     | In progress  | Not yet | [Repo](https://github.com/contentacms/contenta_react) |
| Vue.js + Nuxt.js | In progress  | [Website](https://contentavuenuxt.now.sh) | [Repo](https://github.com/contentacms/contenta_vue_nuxt) |
| Chatbot   | Running demo  | [Blog post](https://medium.com/@dawehner/im-hungry-but-i-can-t-decide-a-small-chatbot-based-upon-contenta-cms-9f8c0bb1d48f) | [Repo](https://github.com/contentacms/contenta-bot) |

If you want to contribute to any of the demo apps above or add a new one, join [the Slack channel](https://drupal.slack.com/messages/C5A70F7D1) and become part of the project!

**Review by e0ipso — CHANGES_REQUESTED** [review]
This is looking great! Some minor changes.

I think that this setup has the potential to replace contentacms.org (https://github.com/contentacms/contentacms.github.io) at some point.

File: docs/development.md (hunk 2 above)
Comment (cjgratacos) [reply docs/development.md]: good point, I was thinking more on readability, yet it prefer not to impact the ability to copy and paste, removing it.

**Review by cjgratacos — COMMENTED** [review]
Comments have been attended

**Review by e0ipso — APPROVED** [review]

**Pull request merged**